package.json

```diff
@@ -1,55 +1,85 @@
 {
   "name": "wallet.universalprofile.cloud",
-  "version": "0.1.0",
+  "packageManager": "yarn@3.6.1",
   "private": true,
+  "type": "module",
   "scripts": {
-    "dev": "next dev",
-    "build": "next build",
-    "start": "next start -p ${PORT:-3000}",
-    "lint": "next lint",
-    "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
-  },
-  "dependencies": {
-    "@erc725/erc725.js": "^0.14.4",
-    "@lukso/lsp-smart-contracts": "^0.6.2",
-    "@types/web3": "^1.2.2",
-    "@walletconnect/client": "^1.8.0",
-    "@walletconnect/qrcode-modal": "^1.8.0",
-    "@walletconnect/types": "^1.8.0",
-    "ethereum-blockies-base64": "^1.0.2",
-    "ethers": "^5.6.9",
-    "next": "12.2.4",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
-    "react-icons": "^4.4.0",
-    "react-toastify": "^9.0.8",
-    "web3": "^1.7.5"
+    "build": "nuxt build",
+    "dev": "nuxt dev",
+    "generate": "nuxt generate",
+    "preview": "nuxt preview",
+    "postinstall": "nuxt prepare",
+    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:*:fix\"",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:{eslint,css,fmt}\"",
+    "lint:eslint": "eslint --ext '.js,.vue,.ts' .",
+    "lint:eslint:fix": "eslint --ext '.js,.vue,.ts' . --fix",
+    "lint:css": "stylelint \"**/*.{scss,css,vue}\"",
+    "lint:css:fix": "stylelint \"**/*.{scss,css,vue}\" --fix",
+    "lint:fmt": "prettier --check --log-level=warn .",
+    "lint:fmt:fix": "prettier --write .",
+    "lint:types": "tsc --noEmit --project tsconfig.json",
+    "prepare": "husky install",
```

Comment:
I don't think we should install Husky.

Happy to discuss this but I think husky adds constraints and anyway, if the CI is correctly set up, we won;t push bad code to the repo.

Having husky adds constraints imo (commits are long to run because the stuff has to run, we can't push WIP commit or quickly change branches with "failing" changes...)...

Reply:
Ok np, I'm happy to remove it. Was re-using setup from migration repo

Reply:
ok super, lets remove it then :)

Reply:
seem like it needs to stay, it's a part of the `lint-staged` installation and also because of what @richtera mentioned:
> With yarn >3 you need to do the yarn husky install manually. Yarn no longer executes postinstall code in dependencies due to security.

Reply:
lint-staged just speeds up dev by not having to wait a few minutes for results in gitops, but other than that they make things more involved when committing.